docs/sphinx/examples/cpp/dynamics/cross_resonance.cpp

```diff
@@ -40,14 +40,14 @@ int main() {
   // operators allow transitions between the spin states (|0> and |1>).
   auto spin_plus = [](int degree) {
     return 0.5 *
-           (cudaq::spin_operator::x(degree) +
-            std::complex<double>(0.0, 1.0) * cudaq::spin_operator::y(degree));
+           (cudaq::spin_handler::x(degree) +
+            std::complex<double>(0.0, 1.0) * cudaq::spin_handler::y(degree));
   };
 
   auto spin_minus = [](int degree) {
     return 0.5 *
-           (cudaq::spin_operator::x(degree) -
-            std::complex<double>(0.0, 1.0) * cudaq::spin_operator::y(degree));
+           (cudaq::spin_handler::x(degree) -
```

Comment:
spin_handler? shouldn't this be spin_op::x?

Reply:
The user should never write code that says "spin_handler". The user facing type is `spin_op`.